CMakeLists.txt

```diff
@@ -1,105 +1,87 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8.5)
+CMake_minimum_required(VERSION 3.0)
 
-project(booredis)
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake")
-string(TOUPPER ${PROJECT_NAME} PROJECT_NAME_UPPER)
+project(booredis VERSION 0.2.0 LANGUAGES CXX)
```

Comment:
Maybe we need a bit more changes for such a release? Do you plan on creating `Booredis::` namespace?

Reply:
Ok, I think we can at least introduce the 'Booredis' C++ namespace afterwards and release it as 0.2.0 eventually. This PR is not merged for too long IMO.